AdminApi/src/AdminApi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.html

```diff
@@ -32,7 +32,12 @@ <h2 class="header-title">{{ header }}</h2>
                         <span matListItemTitle>Tier</span>
                         <span matListItemLine
                             ><mat-chip-listbox aria-label="Tier selection" selectable="false">
-                                <mat-chip-option *ngFor="let t of tiers" [selected]="t == updatedTier" (click)="updatedTier = t" [color]="t.name == 'Basic' ? 'accent' : ''"
+                                <mat-chip-option
+                                    *ngFor="let t of tiers"
+                                    [selected]="t == updatedTier"
+                                    (click)="updatedTier = t"
+                                    [color]="t.name == 'Basic' ? 'accent' : ''"
```

Comment:
I always wondered why the Tiers have different colors. Now I see. :) Can you remove this please? I don't think that this makes sense.

Reply:
Sure thing.